scripts/v.db.droptable/v.db.droptable.html

```diff
@@ -36,8 +36,3 @@ <h2>SEE ALSO</h2>
 <h2>AUTHOR</h2>
 
 Markus Neteler
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
Line 31 

`<a href="v.db.update.html">v.db.update</a><br>`

should be 

`<a href="v.db.update.html">v.db.update</a>,`.


Line 34 unnecessary one empty line.

Reply:
Fixed as for `scripts/v.db.addcolumn/v.db.addcolumn.html` (see above)